autogtp/Management.h

```diff
@@ -19,16 +19,19 @@
 #define MANAGEMENT_H
 
 #include <QAtomicInt>
+#include <QFileInfo>
+#include <QList>
 #include <QMutex>
 #include <QString>
-#include <QTextStream>
-#include <QThread>
-#include <QFileInfo>
-#include <QLockFile>
+#include <QStringList>
 #include <QVector>
 #include <chrono>
 #include <stdexcept>
-#include "Worker.h"
+#include "Order.h"
+#include "Result.h"
```

Comment:
I do not like to include a `.h` that I do not need in a `.h` only because all the `.cpp` using the second `.h`  use also the first. I like the include to be explicit. This invalidate a lot of other changes in the `autogtp` code.

Reply:
For some reason, Qt's meta-object compiler complains if `Result` is forward declared instead.